userguide/dailyuse/anbox.rst

```diff
@@ -1,13 +1,15 @@
 Android apps
 ========================
 
-`Anbox <https://anbox.io>`_ is a minimal Android container and compatibility layer that allows you to run Android apps on GNU/Linux operating systems such as UBports.
+`Anbox <https://anbox.io>`_ is a minimal Android container and compatibility layer to run Android apps on GNU/Linux operating systems such as Ubuntu Touch.
 
 .. note::
-    When "host" is used in this document, it refers to another device which you can connect your Ubuntu Touch device to. Your host device must have ``adb`` and ``fastboot`` installed.
+    "Computer" refers to another device you connect your Ubuntu Touch device to via USB.
```

Comment:
Is wireless adb a think on Ubuntu Touch? (Never used it on Android either.)

Reply:
I don't think so ... well, ... maybe you can piece it (or smt like it) together with ssh and port forwarding!? haven't used wireless adb ever. and I don't think it would really offer a lot of benefits for UT, after all we've got a "real" (enough) linux and can use ssh

Reply:
```suggestion
    "Computer" refers to another device you connect your Ubuntu Touch device to (via USB here).
```